content/docs/sidebar.json

```
@@ -128,36 +146,26 @@
           "running-experiments",
           "hydra-composition",
           "comparing-experiments",
+          "visualizing-plots",
           "sharing-experiments",
           "persisting-experiments",
           "cleaning-experiments",
           "checkpoints"
         ]
       },
-      {
-        "slug": "pipelines",
-        "children": ["defining-pipelines"]
-      },
       {
         "slug": "how-to",
         "source": false,
         "children": [
+          "running-dvc-on-windows",
+          "setup-google-drive-remote",
           "stop-tracking-data",
           "update-tracked-data",
           "add-deps-or-outs-to-a-stage",
           "resolve-merge-conflicts",
           "share-a-dvc-cache"
```

Minor (could be a follow-up) in terms of order and page naming:

```suggestion
          "run-dvc-on-windows",
          "stop-tracking-data",
          "update-tracked-data",
          "resolve-merge-conflicts",
          "add-deps-or-outs-to-a-stage",
          "setup-google-drive-remote",
          "share-a-dvc-cache"
```

---

TBH I think they're fine as is. For example, I intentionally put windows and google drive at the top since they are blockers for doing almost anything for those users.

---

OK but (minor)
- should it be (how to) `run-dvc-on-windows` (not running)
- the one for GDrive may be moved in the future again right? Into data management/ remote setup (see https://github.com/iterative/dvc.org/issues/2866)